### PR TITLE
feat: add various links

### PR DESCRIPTION
**Pull request by scriptswhisper**
# Pull Request
Compile all the information written below to send a legitimate pull request to the repository!

### This PR is labeled as:
*Choose only one and remember to select the label even on the right.*

- [X] Feature
- [ ] Bug
- [ ] Other

## Solution
- Added steam deck refurbished
- Best accessories recommended by XDA
- Added two interesting software for third-party tools and graphical boot manager
- Moved deck verified from utility to general website
- Added r/WindowsOnDeck subreddit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +72,7 @@ Choose what you find more suitable for you:
 - [Extreme Rate Shells](https://extremerate.com/collections/steam-deck-shells) - Extreme Rate Steam Deck Shells.
 - [GamingCobra Shells](https://www.gamingcobra.com/collections/steam-deck-shells) - Gaming Cobra Steam Deck Shells.
 - [JSAUX Transparent Back Plate](https://jsaux.com/products/transparent-back-plate-for-steam-deck-pc0106) - JSAUX Transparent Back Plate for Steam Deck.
+- [Best Xda Accessories](https://www.xda-developers.com/best-steam-deck-accessories/) - Best Steam Deck accessories by XDA.
```
Comment (airscripts) [inline README.md:75]: What do you think about exploding this link with the list inside of it?  
I can help you with this if you need.

File: README.md [hunk 2]
```diff
@@ -79,6 +81,8 @@ Choose what you find more suitable for you:
 - [Steam ROM Manager](https://github.com/SteamGridDB/steam-rom-manager) - An app for managing ROMs in Steam.
 - [Discord Overlay](https://trigg.github.io/Discover/deckaddnonsteamgame) - Discord Overlay for Steam.
 - [SteamOS Btrfs](https://gitlab.com/popsulfr/steamos-btrfs) - Convert the /home partition from ext4 to btrfs.
+- [ProtonUp-Qt](https://davidotek.github.io/protonup-qt/) - Install and manage third-party compatibility tools for Steam Geames.
```
Comment (airscripts) [inline README.md:84]: Typo here.

```suggestion
- [ProtonUp-Qt](https://davidotek.github.io/protonup-qt/) - Install and manage third-party compatibility tools for Steam Games.
```

**Review by airscripts — COMMENTED** [review]
Really good work.  
I've asked you a few things.

File: README.md (hunk 2 above)
Comment (scriptswhisper) [reply README.md:84]: thanks, my bad!

File: README.md (hunk 1 above)
Comment (scriptswhisper) [reply README.md:75]: Don't worry, I wasn't sure about the list anyway, I was waiting for your feedback.!

Comment (scriptswhisper) [reply README.md]: Updated

File: README.md [hunk 3]
```diff
@@ -71,6 +72,18 @@ Choose what you find more suitable for you:
 - [Extreme Rate Shells](https://extremerate.com/collections/steam-deck-shells) - Extreme Rate Steam Deck Shells.
 - [GamingCobra Shells](https://www.gamingcobra.com/collections/steam-deck-shells) - Gaming Cobra Steam Deck Shells.
 - [JSAUX Transparent Back Plate](https://jsaux.com/products/transparent-back-plate-for-steam-deck-pc0106) - JSAUX Transparent Back Plate for Steam Deck.
+- [Corsair K70 Pro Mini](https://www.amazon.com/dp/B0B4SW81GW/) - 60% wireless mechanical keyboard.
+- [Razer Viper V2 Pro](https://www.amazon.it/dp/B09QFVSYX6) - Premium gaming mouse with great set of features.
```
Comment (airscripts) [inline README.md:76]: I've tried the .com link for this and seems to be linking towards the same product.
The only thing that concerns me is that is currently inavailable.
Do you want to keep it with a .com link or remove it?

**Review by airscripts — APPROVED** [review]
Nicely done!

File: README.md (hunk 3 above)
Comment (scriptswhisper) [reply README.md]: Updated with another link of amazon.com

**Pull request merged**